File: development/NET/samples/server/ServerDLL/VB/OpcProject/MyRequest.vb
Imports System
Imports Softing.OPCToolbox.Server

Namespace DllSample

	Public Class MyRequest
		Inherits DaRequest

		Public Sub New(ByVal transactionType As EnumTransactionType, ByVal sessionHandle As UInt32, ByVal aDaAddressSpaceElement As DaAddressSpaceElement, ByVal propertyID As Integer, ByVal requestHandle As UInt32)
			MyBase.New(transactionType, sessionHandle, aDaAddressSpaceElement, propertyID, requestHandle)
		End Sub
	End Class
End Namespace

File: development/NET/samples/server/ServerDLL/CS/OpcProject/MyRequest.cs
using Softing.OPCToolbox.Server;

namespace DllSample
{
	public class MyRequest : DaRequest
	{
		#region Constructor
		//-----------------

		public MyRequest(
			EnumTransactionType transactionType,
			uint sessionHandle,
			DaAddressSpaceElement aDaAddressSpaceElement,
			int propertyID,
			uint requestHandle)
			:
			base(
				transactionType,
				sessionHandle,
				aDaAddressSpaceElement,
				propertyID,
				requestHandle)
		{

		}   //	end constructor

		//--
		#endregion

	}   //	end MyRequest class
}   //	ens namespace
